### PR TITLE
daisy: Add StopStep

### PR DESCRIPTION
**Pull request by helen-fornazier**
Step to stop an instance without deleting it.
The StopStep is similar to DeleteResources for instances, but allow
adding WaitForInstanceSignal without making it a dependency to the
Delete step.

Depend on #396

### CODE REVIEW

File: daisy/step_stop_instances.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	w := s.w
+	e := make(chan dErr)
+
+	for _, i := range st.Instances {
+		wg.Add(1)
+		go func(i string) {
+			defer wg.Done()
+			w.Logger.StepInfo(w, "StopInstances", "stopping instance %q.", i)
+			if err := w.instances.stop(i); err != nil {
+				if err.Type() == resourceDNEError {
```
Comment (adjackura) [inline daisy/step_stop_instances.go]: This should never happen, just go ahead and remove the resourceDNEError check and log

File: daisy/compute/compute.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ type Client interface {
 	DeleteDisk(project, zone, name string) error
 	DeleteImage(project, name string) error
 	DeleteInstance(project, zone, name string) error
+	StopInstance(project, zone, name string) error
```
Comment (adjackura) [inline daisy/compute/compute.go]: You'll also need to add the test implementation to test_client, thats probably why the tests are failing

File: daisy/step.go [hunk 3]
```diff
@@ -48,6 +48,7 @@ type Step struct {
 	CreateInstances        *CreateInstances        `json:",omitempty"`
 	CreateNetworks         *CreateNetworks         `json:",omitempty"`
 	CopyGCSObjects         *CopyGCSObjects         `json:",omitempty"`
+	StopInstances          *StopInstances          `json:",omitempty"`
```
Comment (adjackura) [inline daisy/step.go]: Would you also need startinstances? If you don't need it for you tests there's no need to add it now, though we proabbly should make an issue to track it.

Comment (helen-fornazier) [reply daisy/step.go]: I don't need startinstances for my tests, I'll make an issue then

File: daisy/compute/compute.go (hunk 2 above)
Comment (helen-fornazier) [reply daisy/compute/compute.go]: done, could you check please?

**Comment by helen-fornazier** [comment]
sorry, codecov wasn't configured in my repo, I'll fix those checks and update the PR

**Pull request merged**